packages/expo-dev-launcher/src/DevLauncher.ts

```diff
@@ -2,21 +2,32 @@ import { ExpoUpdatesManifest } from 'expo-manifests';
 import { NativeModules } from 'react-native';
 import './setUpErrorHandler.fx';
 
-// Dev launcher manifests are only ones served by servers (not embedded bare manifests)
+/**
+ * @hidden Dev launcher manifests are only ones served by servers (not embedded bare manifests)
+ */
 export type Manifest = ExpoUpdatesManifest;
 
 export { disableErrorHandling } from './DevLauncherErrorManager';
 
+/**
+ * @hidden
+ */
 export function registerErrorHandlers() {
   console.warn(
     'DevLauncher.registerErrorHandlers has been deprecated. To enable error handlers you need to import "expo-dev-launcher" at the top of your index.js.'
   );
 }
 
+/**
+ * A method returning a boolean indicating if the current application is a development build.
```

Comment:
```suggestion
 * A method that returns a boolean to indicate if the current application is a development build.
```

Let's update the verbiage to keep the language easier to read.